pywbem/cim_operations.py

```diff
@@ -453,11 +459,33 @@ class WBEMConnection(object):
         :class:`~pywbem.BaseOperationRecorder`, each operation
         that is executed on this connection will be recorded by
         invoking its :meth:`~pywbem.BaseOperationRecorder.record` method.
+
+      use_pull_operations (:class:`py:bool`):
+        Indicates whether the client will attempt the use of pull operations in
+        any `Iter...()` methods.
+
+        This variable is initially set from the same-named input argument, and
+        is modified when `Iter...()` methods are invoked, dependent on the
+        support for pull operations in the WBEM server.
+        After modifying the variable, it is either `True` or `False` and
+        indicates whether pull operations will be used in any subsequent
+        `Iter...()` methods.
+
```

Comment:
The new text for the `use_pull_operations` instance variable (above this comment) makes the part of the old text that is till below this comment (lines 474 to 482) superfluous. They should be removed.